# Agent id
uint8 id

# Agent position
#float64 x
#float64 y

# Agent velocity
#float64 vx
#float64 vy

# Agent current goal
#float64 gx
#float64 gy

# Cells covered by agent
coverage_control/SensingInfo[] covered_cells